src/core/annotation.js

```diff
@@ -704,6 +708,46 @@ class Annotation {
     }
   }
 
+  setRotation(mk) {
+    this.rotation = 0;
+    if (mk instanceof Dict) {
+      let angle = mk.get("R") || 0;
+      if (angle !== 0) {
+        // If the angle is negative then angle % 360 is still negative.
+        angle = (360 + (angle % 360)) % 360;
```

Comment:
I'm finding this a bit less clear than what we have in other parts of the code-base, see e.g.

https://github.com/mozilla/pdf.js/blob/45de73bd00d56c6f0df2edac98eccf51c7ca42ca/src/display/display_utils.js#L179-L183 and https://github.com/mozilla/pdf.js/blob/45de73bd00d56c6f0df2edac98eccf51c7ca42ca/web/base_viewer.js#L509-L513